### PR TITLE
config: add shared block cache config

### PR DESCRIPTION
**Pull request by yiwu-arbug**
Update tikv config template to reflect shared block cache change in https://github.com/tikv/tikv/pull/4563

Tested by deploy and start a cluster.

### CODE REVIEW

**Comment by siddontang** [comment]
seem you don't change grafana here?

**Comment by yiwu-arbug** [comment]
@siddontang I'm not changing Grafana for now. Grafana will still show block cache size but show the same size for all CFs. It's probably fine I think.

File: roles/tikv/templates/tikv.toml.j2 [hunk 1]
```diff
@@ -30,7 +30,15 @@
 
 [storage]
 {% for item, value in tikv_conf.storage | dictsort -%}
+{% if value is not mapping -%}
 {{ item }} = {{ value | to_json }}
+{% else %}
+
+[storage.{{ item }}]
+{% for sub_item, sub_value in value | dictsort -%}
+{{ item }} = {{ value | to_json }}
```
Comment (Connor1996) [inline roles/tikv/templates/tikv.toml.j2]: I'm not familiar with it, but I think it should be `{{ sub_item }} = {{ sub_value | to_json }}` ?

**Comment by zhangjinpeng1987** [comment]
I think the block-cache expression in scripts/tikv_details.json and scripts/tikv_trouble_shoot.json also need changing.

File: roles/tikv/templates/tikv.toml.j2 (hunk 1 above)
Comment (yiwu-arbug) [reply roles/tikv/templates/tikv.toml.j2]: thanks!

**Comment by yiwu-arbug** [comment]
@zhangjinpeng1987 I did not change the block cache metrics in the other PR.

**Pull request merged**